packages/mdc-floating-label/_mixins.scss

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "./variables";
+
+// Public
+
+@mixin mdc-floating-label-color($color) {
+  .mdc-floating-label {
+    @include mdc-theme-prop(color, $color);
+  }
+}
+
+// Private
+
+@mixin mdc-floating-label-invalid-shake-keyframes_($modifier, $positionY, $positionX: 0%, $scale: .75) {
```

Comment:
I think it might make more sense to make this public, and move the @includes to the text-field scss file, along with the variables. Reason being select might not have the same variants (box, outlined, outlined-dense, etc). 

Want to see what reviewer thinks before doing this.